modules/dists/BlockDist.chpl

```diff
@@ -1125,9 +1126,10 @@ iter BlockArr.these(param tag: iterKind, followThis, param fast: bool = false) r
 
   for param i in 1..rank {
     var stride = dom.whole.dim(i).stride;
+    var alignment = whole.dim(i).alignment;
```

Comment:
D'oh!  Yes.  You can tell I wanted to go to bed too badly to test that branch (I was proud to have even looked for it after getting the original test to work.  Did I mention I'd been up since 3am yesterday when I wrote this at 11:30pm?